src/Task.ts

```diff
@@ -41,6 +43,32 @@ export function handlePriorityFeeLimitSet(event: PriorityFeeLimitSet): void {
   task.save()
 }
 
+export function handleTokensAcceptanceTypeSet(event: TokensAcceptanceTypeSet): void {
+  const task = Task.load(event.address.toHexString())
+  if (task == null) return log.warning('Missing task entity {}', [event.address.toHexString()])
+
+  const acceptanceType = parseAcceptanceType(event.params.acceptanceType)
+  const acceptanceListId = task.id
+  const acceptanceList = loadOrCreateTokenAcceptanceList(acceptanceListId)
```

Comment:
It's more readable when you define a variable right before you use it:

```suggestion
  const acceptanceListId = task.id
  const acceptanceList = loadOrCreateTokenAcceptanceList(acceptanceListId)
  const acceptanceType = parseAcceptanceType(event.params.acceptanceType)
```